requests/utils.py

```diff
@@ -820,7 +821,7 @@ def default_headers():
     """
     return CaseInsensitiveDict({
         'User-Agent': default_user_agent(),
-        'Accept-Encoding': ', '.join(('gzip', 'deflate')),
+        'Accept-Encoding': make_headers(accept_encoding=True)["accept-encoding"],
```

Comment:
Should we generate this header once and store the result so it doesn't need to be called every time? The value is determined by imports so shouldn't be changing often.

Reply:
This optimization makes sense.  I kindly ask you to implement it in the way, you think fits best.